app/src/main/res/layout/comment_composer_view.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:background="@color/white"
+    android:padding="@dimen/grid_3">
+
+    <ImageView
+        android:id="@+id/avatar"
+        android:layout_width="@dimen/comments_feed_avatar_width"
+        android:layout_height="@dimen/comments_feed_avatar_height"
+        android:layout_alignParentStart="true"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        tools:ignore="ContentDescription" />
+
+    <View
+        android:id="@+id/editTextTextBackground"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:layout_marginStart="@dimen/grid_2"
+        android:background="@drawable/border_rounded_soft_grey_50"
+        app:layout_constraintBottom_toBottomOf="@+id/comment_text_composer"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toEndOf="@+id/avatar"
+        app:layout_constraintTop_toTopOf="@+id/comment_text_composer" />
+
+    <com.google.android.material.textfield.TextInputEditText
+        android:id="@+id/comment_text_composer"
+        style="@style/CommentsTextInputLayout"
+        android:hint="Leave a comment…"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toStartOf="@+id/comment_action_button"
+        app:layout_constraintStart_toStartOf="@+id/editTextTextBackground"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <androidx.appcompat.widget.AppCompatButton
+        android:id="@+id/comment_action_button"
+        style="@style/CommentsButton"
+        android:enabled="false"
+        android:text="@string/social_buttons_send"
+        android:visibility="gone"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="@+id/editTextTextBackground"
+        app:layout_constraintStart_toEndOf="@+id/comment_text_composer" />
+
+    <androidx.constraintlayout.widget.Group
+        android:id="@+id/comment_text_group"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        app:constraint_referenced_ids="editTextTextBackground,comment_text_composer" />
+
+    <androidx.appcompat.widget.AppCompatTextView
+        android:id="@+id/comments_disable_msg"
+        style="@style/CommentsDisabledMessageTextView"
+        android:text="Only backers can leave comments"
```

Comment:
For the texts you can add a placeholder string on `strings.xml` (https://github.com/kickstarter/android-oss/blob/master/app/src/main/res/values/strings.xml) until we get the translations

Reply:
Done

Reply:
for the check its on style and tested 
![image](https://user-images.githubusercontent.com/1075310/118798735-4496a900-b89e-11eb-9d98-8316e03741ce.png)